.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+<!--
+If you are asking a question rather than filing a bug, try one of these instead:
```

Comment:
something like "you'll get better results using one of these instead" will probably convince a greater number of people